docs/ota-client-guide/antora.yml

```diff
@@ -1,6 +1,6 @@
 name: ota-client
 title: OTA Connect Developer Guide
 version: latest
-display_version: 2020.3 (latest)
+display_version: 2020.5 (latest)
```

Comment:
Too soon? I know I need to change the 2020.4-docs branch once I actually make the release.